docs/source/eventing/actions/label.rst

```diff
@@ -20,10 +20,10 @@ Changes the text and color of a generic label device.
 +----------+------------------+---------------------+---------------------------------------------------+
 | TO       | |yes|            | |no|                | New text                                          |
 +----------+------------------+---------------------+---------------------------------------------------+
-| COLOR    | |no|             | |no|                | New color. If omitted, color is set to black"     |
-+----------+------------------+---------------------+---------------------------------------------------+
 | **Name** | **Required**     | **Multiple Values** | **Description**                                   |
 +----------+------------------+---------------------+---------------------------------------------------+
+| COLOR    | |no|             | |no|                | New color                                         |
```

Comment:
1. Why was this changed?
2. If needed, can you make this a seperate PR?

Reply:
I only intended to correct a typo (the trailing double quotes) ....
Shall I revert it, including the typo and make a PR to correct that later?

Reply:
Yes, make it a seperate PR. And then again, you didn't just remove the trailing quote but removed a large part of that line.

Reply:
Reverted deleted text